test/SlowTests/Cluster/ClusterIndexNotificationsTest.cs

```diff
@@ -93,6 +95,22 @@ public async Task ShouldThrowTimeoutException()
             }
         }
 
+        [Fact]
+        public async Task RavenDB_14086()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var indexes = new List<Task>();
+                for (int i = 0; i < 100; i++)
+                {
+                    var index = new Index();
+                    indexes.Add(index.ExecuteAsync(store));
+                }
+
+                await Task.WhenAll(indexes);
+            }
```

Comment:
Don't you need to verify that they all exist here?

Reply:
done